_posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc

```diff
@@ -589,16 +588,16 @@ Use the link:https://www.terraform.io/docs/providers/aws/r/cloudtrail.html[`aws_
 is_multi_region_trail         = true
 include_global_service_events = true
 enable_log_file_validation    = true
-s3_bucket_name                = <YOUR BUCKET NAME>
-cloud_watch_logs_group_arn    = <YOUR CLOUDWATCH LOGS GROUP ARN>
+s3_bucket_name                = local.cloudtrail_s3_bucket_name
```

Comment:
Reading [this section at the guide](https://deploy-preview-441--keen-clarke-470db9.netlify.app/guides/compliance/how-to-achieve-cis-benchmark-compliance/#logging-2), this `local` seems a bit  out of place, it is not connected to the Landing Zone steps so I think that ` cloud_watch_logs_group_arn    = <YOUR CLOUDWATCH LOGS GROUP ARN>` are more readable here.

Reply:
ok, changing it now! 🙇